system/shthreads.py

```diff
@@ -261,9 +261,16 @@ def status(self):
         # False     | not None    | stopped
         # True      |     None    | impossible
         # True      | not None    | running
-        if self.isAlive():
+        
+        # `Thread.isAlive()` was removed in Python 3.9
+        if hasattr(self, 'is_alive'):
+            is_alive = self.is_alive()
+        else:
+            is_alive = self.isAlive()
+    
+        if is_alive:
```

Comment:
`Thread.is_alive()` was added in ___Python 2.6___ so we can merely change `.isAlive()` to `.is_alive()` everywhere.
* https://docs.python.org/2/library/threading.html#threading.Thread.is_alive
```suggestion
        if self.is_alive():
```